### PR TITLE
Several fixes in formatting

### PR DESCRIPTION
**Pull request by moxiegirl**
- fixing headings in run.md
- creating a table for readability
- adding index for logging
- moving logging overview into logging

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @docker/docs-owners

File: docs/reference/logging/fluentd.md [hunk 1]
```diff
@@ -25,8 +25,16 @@ driver sends the following metadata in the structured log message:
 | `container_name` | The container name at the time it was started. If you use `docker rename` to rename a container, the new name is not reflected in the journal entries.                                         |
 | `source`         | `stdout` or `stderr`                |
 
+The `docker logs` command is not available for this logging driver.
+
 ## Usage
 
+Some options are supported by specifying `--log-opt` as many as needed:
```
Comment (thaJeztah) [inline docs/reference/logging/fluentd.md]: s/as many/as many times/ ?

File: docs/reference/logging/fluentd.md [hunk 2]
```diff
@@ -25,8 +25,16 @@ driver sends the following metadata in the structured log message:
...
+
+ - `fluentd-address`: specify `host:port` to connect [localhost:24224]
```
Comment (thaJeztah) [inline docs/reference/logging/fluentd.md]: not sure about the square brackets here, should this just be (for example, `localhost:24224`)

File: docs/reference/logging/fluentd.md [hunk 3]
```diff
@@ -25,8 +25,16 @@ driver sends the following metadata in the structured log message:
...
+ - `fluentd-tag`: specify tag for fluentd message, which interpret some markup, ex `{{.ID}}`, `{{.FullID}}` or `{{.Name}}` [docker.{{.ID}}]
```
Comment (thaJeztah) [inline docs/reference/logging/fluentd.md]: same (square brackets)

File: docs/reference/logging/overview.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+|-------------|-------------------------------------------------------------------------------------------------------------------------------|
+| `json-file` | Default logging driver for Docker. Writes JSON messages to file.                                                              |
+| `syslog`    | Syslog logging driver for Docker. Writes log messages to syslog.                                                              |
+| `journald`  | Journald logging driver for Docker. Writes log messages to `journald`.                                                        |
+| `gelf`      | Graylog Extended Log Format (GELF) logging driver for Docker. Writes log messages to a GELF endpoint likeGraylog or Logstash. |
+| `fluentd`   | Fluentd logging driver for Docker. Writes log messages to `fluentd` (forward input).                                          |
+
+The `docker logs`command is available only for the `json-file` logging driver.  
+
+### The json-file options
```
Comment (thaJeztah) [inline docs/reference/logging/overview.md]: I was actually thinking to split this page up and give each driver a separate page, with the same structure and explaining the options for that driver on the separate page.

The JSON-file, syslog, journald, Gelf, and Fluentd options below would then be on separate pages.

This file could then become the index again, containing only the table. wdyt?

File: docs/reference/run.md [hunk 5]
```diff
@@ -507,16 +505,18 @@ parent group.
...
+| `-c`, `--cpu-shares=0`               | CPU shares (relative weight)                                                                |
+| `--cpu-period=0`                     | Limit the CPU CFS (Completely Fair Scheduler) period                                        |
+| `--cpuset-cpus="" `                  | CPUs in which to allow execution (0-3, 0,1)                                                 |
+| `--cpuset-mems=""`                   | Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems. |
+| `--cpu-quota=0`                      | Limit the CPU CFS (Completely Fair Scheduler) quota                                         |
+| `--blkio-weight=0`                   | Block IO weight (relative weight) accepts a weight value between 10 and 1000.               |
+| `--oom-kill-disable=true` or `false` | Whether to disable OOM Killer for the container or not.                                     |
+| `--memory-swappiness=""  `           | Tune a container's memory swappiness behavior. Accepts an integer between 0 and 100.        |
```
Comment (thaJeztah) [inline docs/reference/run.md]: <3 for the table. Should we use 4 columns; *Option*, *Type* (Boolean, Int, ...), *Default value*, and *Description*? Or would that be too much detail?

**Comment by thaJeztah** [comment]
ping @moxiegirl - what are your thoughts on https://github.com/docker/docker/pull/15102#discussion_r35702648?

**Comment by moxiegirl** [comment]
@thaJeztah This PR is a start on exactly what you are suggesting.  Notice that  `fluentd` and `journald` already have their own pages.  Breaking out the drivers is not something I have in my scope.  So, we will need to be iterative. 

For now, the purpose of the index.md is to form the "Logging Drivers" menu item.  Menu items with submenus right now aren't "clickable" in the documentation design --- they are just labels.  I"m trying to get that changed but for now it is a limitation we have to live with.

**Comment by thaJeztah** [comment]
@moxiegirl okay, on doing this iterative. I'll add my LGTM here, but I'll leave the merge to you, so that you can decide if you want to address some of my other nits as part of this PR (or keep them for later)

:+1:

File: docs/reference/run.md (hunk 5 above)
Comment (moxiegirl) [inline docs/reference/run.md]: Let's leave this as it is.  First, if we were to do this format it begs why we don't use similar format in other similar situations.  Also, a drawback of authoring in Markdown is that it doesn't make managing table layout very easy --- especially with more detailed tables.

**Comment by moxiegirl** [comment]
Author LGTM

**Comment by moxiegirl** [comment]
@calavera cherry pick please

@thaJeztah Thanks for the comments Seb.

File: docs/reference/run.md (hunk 5 above)
Comment (thaJeztah) [inline docs/reference/run.md]: Not being able to wrap is a real pita yup

**Pull request merged**